src/components/home/PanelResult.vue

```diff
@@ -56,28 +56,47 @@
           </template>
         </help-button>
       </v-card-title>
-      <v-card-text class="body-1">
+      <v-card-text>
         <info-alert :active="help">
           <template v-slot:content>
             <panel-results-help />
           </template>
         </info-alert>
-        <span v-if="userGenes.length > 0">
-          {{ $t('panel-result.result.title') }}: {{ userGenes.length }}
-        </span>
+        <v-container v-if="userGenes.length > 0 || notFoundGenes.length > 0">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="6"
+              md="4"
+              offset-md="4"
+            >
+              <v-simple-table class>
```

Comment:
I'm not too crazy about this table. It would be better I think to just have 2 lines of text aligned left (2 div) instead. This way you also won't need to repeat the v-if twice.

Reply:
done